ARCs/arc-0003.md

```diff
@@ -92,13 +96,34 @@ The JSON Metadata schema is as follows:
 
 The property `decimals` is **OPTIONAL**. If provided, it **MUST** match the ASA parameter `dt`.
 
+#### Integrity Properties
+
+Compared to ERC-1155, the metadata JSON schema allows to indicate digests of the files pointed by any URI field.
+This is to ensure the integrity of all the files referenced by the ASA.
+Concretly, every URI property `xxx` is allowed to have an optional associated property  `xxx_integrity` that specifies the digest of the file pointed by the URI.
+
+The digests are represented as a single SHA-256 integrity metadata as defined in the [W3C subresource integrity specification](https://w3c.github.io/webappsec-subresource-integrity).
+Details on how to generate those digests can be found on the [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) (where `sha384` or `384` are to be replaced by `sha256` and `256` respectively as only SHA-256 is supported by this ARC).
+
+It is **RECOMMENDED** to specify all the `xxx_integrity` properties of all the `xxx` URI properties.
```

Comment:
DOn't we say **SHOULD** in these?

Reply:
RECOMMENDED and SHOULD have the same meaning in RFC: https://datatracker.ietf.org/doc/html/rfc2119